BinaryObjectScanner/Utilities/FileTypes.cs

```diff
@@ -259,6 +259,20 @@ public static SupportedFileType GetFileType(byte[] magic)
 
             #endregion
 
+            #region RealArcade
+
+            // RASGI2.0
+            // Found in the ".rgs files in IA item "Nova_RealArcadeCD_USA".
+            if (magic.StartsWith(new byte?[] { 0x52, 0x41, 0x53, 0x47, 0x49, 0x32, 0x2E, 0x30 }))
+                return SupportedFileType.RealArcadeInstaller;
+
+            // XZip2.0
+            // Found in the ".mez" files in IA item "Nova_RealArcadeCD_USA".
+            if (magic.StartsWith(new byte?[] { 0x58, 0x5A, 0x69, 0x70, 0x32, 0x2E, 0x30 }))
+                return SupportedFileType.RealArcadeMezzanine;
```

Comment:
Do these need full models? Or is just the magic number known?

Reply:
Full models would be ideal at some point, and would allow for better parsing of data such as game ID (which is present in plaintext toward the start of the file), but isn't really needed by any means.